etc/benchmarks/lib_runner.mjs

```diff
@@ -52,15 +54,15 @@ export function getCurrentLocalBSON(libs) {
 
 export async function getLibs() {
   return await Promise.all([
-    (async () => {
-      const { stdout } = await exec('git rev-parse --short HEAD');
-      const hash = stdout.trim();
-      return {
-        name: 'local',
-        lib: await import('../../lib/bson.js'),
-        version: hash
-      };
-    })(),
+    // (async () => {
```

Comment:
if we're going to keep commented code, we should add a description of its purpose (same goes for the code in L78)

Reply:
Uncommented importing the current local BSON, and commented on BSON-EXT